system/lib/websocket/websocket_to_posix_socket.cpp

```diff
@@ -21,6 +22,14 @@
 extern "C"
 {
 
+in_addr_t inet_addr(const char *cp)
+{
+	struct in_addr val;
+	if (inet_aton(cp, &val))
+		return val.s_addr;
+	return INADDR_NONE;
+}
+
```

Comment:
This looks odd, why implement `inet_addr` here? It looks like it is provided by sockets library, declared in that `<arpa/inet.h>`